source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,105 @@
+* You have to add the `role="button"` attribute to classify it as a button.
+* You have to add a `tabIndex` which will allow the `div` to be navigated to using only the `tab` key
+* You have to add handling for the space and enter key that press the button
+
+{% pullquote %}
+A list of role attribute values can be found on [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Composite_roles).
+{% endpullquote %}
+
+After adding roles to the controls and content in your webpage, the next thing to look over are `aria` attributes. For instance, we use `aria-live` for our `ProgressBar` slider. This is because the `ProgressBar` is always updating when a video is playing.
+
+{% pullquote %}
+For a more complete list of [ARIA attributes see the specification](https://www.w3.org/TR/wai-aria-1.1/).
+{% endpullquote %}
+
+In video "the action, from which the status can be inferred" covers both bases and describes the status quo
+
+Finally you need to add an accessible "name" to an element so that it can be referred to correctly. For instance to indicate that the `MuteToggle` is not just a "button" we have to set the `title` and the `innerHTML`/`innerText` to "Mute Toggle". In Video.js we refer to the accessible name and the action that a control performs as "control text". When the action the a control performs changes so does the control text. This will allow the screen reader to refer to the `MuteToggle` as "Mute Toggle" rather than "button". It will also update the "Mute Toggle" action to be defined as "Mute" or "Unmute" depending on what the button would do when pressed (ie the state of the button).
+
+Here are some examples of accessiblity straight from Video.js:
+* The `MuteToggle` `<button>`:
+  * Has `aria-live` set to `polite`, which means that the screen reader should wait until the user is idle to send `aria-valuenow` updates to the user
+  * Has control text of "Mute" or "Unmute" which indicates the current status of the button to the user
+* The `VolumeBar` slider `<div>`:
+  * Has a `role` attribute with a value of `slider` like this: `role="slider"`
+  * Has a `tabIndex` attribute
+  * Has `EventHandlers` that listen for:
+    * The up and right arrow keys to increase the volume and the slider percentage
+    * The down and left arrow keys to decrease the volume and the slider percentage
+  * Has `aria-label` of "volume level" which is an accessible name that the screen reader should refer to this with
+  * Has `aria-valuenow` and `aria-valuetext` properties that update to indicate the current volume level (so the screen reader can read it)
+  * Has `aria-live` set to `polite`, which means that the screen reader should wait until the user is idle to send `aria-valuenow` updates to the user
```

Comment:
Again, `aria-live` is set to `polite` (instead of defaulting to `off`) so that a screen reader will announce a new value when a change happens because the user pressed a key, without having to move focus away and back to the control.